src/interfaces/ConfigStore.ts

```diff
@@ -3,8 +3,7 @@ import { RateModelDictionary } from "../lpFeeCalculator/rateModel";
 import { SortableEvent } from "./Common";
 
 export interface ParsedTokenConfig {
-  transferThreshold: string;
-  rateModel: RateModelDictionary;
+  rateModel: across.rateModel.RateModelDictionary;
```

Comment:
I suspect this might be a bad merge - we don't import `across` from the UMA SDK anymore.

```suggestion
  rateModel: RateModelDictionary;
```